cmd/node/main.go

```diff
@@ -701,6 +702,11 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		return err
 	}
 
+	softwareVersionChecker, err := softwareVersion.NewSoftwareVersionChecker(coreComponents.StatusHandler)
```

Comment:
can't we have a factory for all these metrics ?

Reply:
+1

Reply:
done.